etl/steps/export/multidim/health/latest/vaccination_coverage.py

```diff
@@ -24,6 +24,7 @@ def run() -> None:
         indicator_names=["coverage", "unvaccinated", "vaccinated"],
         dimensions=["antigen"],
         indicators_slug="metric",
+        indicator_as_dimension=True,
```

Comment:
totally fine to keep, but not needed

Reply:
Oh, right, I think it may not be necessary after reverting changes (it may have been necessary while I was using the alternative `expand_config` function for some reason).

Reply:
I've reverted it (nothing changed in mdim diff).